sourcecode/apis/contentauthor/app/Libraries/NDLA/Importers/Handlers/APIArticle/Tags.php

```diff
@@ -15,18 +15,6 @@ public function process(Article $article, $jsonArticle): Article
 
         $tags = $this->jsonArticle->tags->tags ?? [];
```

Comment:
Unused variable. Is this class necessary?

Reply:
Removed now, although it probably wasn't related to metadataservice.